src/odoc/compile.ml

```diff
@@ -79,7 +85,7 @@ let resolve_parent_page resolver f =
   in
   let extract_parent = function
     | { Paths.Identifier.iv = `Page _; _ } as container -> Ok container
-    | _ -> Error (`Msg "Specified parent is not a parent of this file")
+    | { Paths.Identifier.iv = `LeafPage _; _ } -> Error (`Msg "leaf page")
```

Comment:
Does this error message ever get exposed to the user?

Reply:
No, this is a leftover from debugging. The more exhaustive matching should be kept though.

```suggestion
    | { Paths.Identifier.iv = `LeafPage _; _ } -> Error (`Msg "Specified parent is not a parent of this file")
```